truffle-config.js

```diff
@@ -65,7 +71,7 @@ module.exports = {
         eth_mainnet: {
             provider: () =>
                 new HDWalletProvider(
-                    mnemonic,
+                    this.secrets.mnemonic,
                     "https://mainnet.infura.io/v3/cf38c21326954ac28aa4f8c3ee33550c"
```

Comment:
Why our infura id is used in the URL ?

Reply:
That was present in truffle-config when I branched off. I don't remember who added it here.
But I can change that to read values from `.secrets.json`